oarepo_requests/ui/theme/assets/semantic-ui/js/oarepo_requests_ui/record-requests/components/RequestListContainer.jsx

```diff
@@ -57,12 +57,12 @@ export const RequestListContainer = ({ requestTypes, isLoading, loadingError, fe
       <SegmentGroupOrEmpty>
         <Segment className="requests-my-requests">
           <Header size="small" className="detail-sidebar-header">{i18next.t("My Requests")}</Header>
-          {!_isEmpty(otherRequests) ? <RequestList requests={otherRequests} requestTypes={requestTypes} fetchNewRequests={fetchNewRequests} /> : <p>{i18next.t("No requests to show")}.</p>}
+          {!_isEmpty(otherRequests) ? <RequestList requests={otherRequests} requestTypes={requestTypes} /> : <p>{i18next.t("No requests to show")}.</p>}
         </Segment>
         {requestsToApprove.length > 0 && (
           <Segment className="requests-requests-to-approve">
             <Header size="small" className="detail-sidebar-header">{i18next.t("Requests to Approve")}</Header>
-            <RequestList requests={requestsToApprove} requestTypes={requestTypes} requestModalType="accept" fetchNewRequests={fetchNewRequests} />
+            <RequestList requests={requestsToApprove} requestTypes={requestTypes} requestModalType="accept" />
```

Comment:
RequestList should pull itself everything it needs from context, avoid prop drilling